### DISCUSSION TITLE
Single integrated report for image, fs, sbom, config

### DISCUSSION THREAD

**Discussion by yreddy-CS** [fatihtokus/scan2html#90] [General]
Is there a way to output all reports(image, sbom,fs, config) into a single HTML file instead of haivng to upload each json

**Comment by fatihtokus** [comment]
Hi @yreddy-CS ,

Thank you for your interest in the plugin.  If you create one report via one trivy command then you will have all in one place. (i.e. trivy scan2html fs --scanners vuln,secret,misconfig . interactive_report.html ) But if you need to execute multiple trivy commands then unfortunately, uploading each json is the only option.

Please consider giving us a GitHub star ⭐️. Thank you!

Regards,
Fatih

**Reply by fatihtokus** [reply]
@yreddy-CS, good news for you. this feature is released. Please try and let me know what you think.

The usage: trivy scan2html generate single_unified_report.html from sbom.json k8s.json ...
